napari_aicsimageio/core.py

```diff
@@ -93,8 +93,13 @@ def _get_meta(data: xr.DataArray, img: AICSImage) -> Dict[str, Any]:
         meta["scale"] = tuple(scale)
 
     # Apply all other metadata
-    meta["metadata"] = {"ome_types": img.metadata}
+    img_meta = {"aicsimage": img, "raw": img.metadata}
+    try:
+        img_meta["ome_types"] = img.ome_metadata
```

Comment:
remind me what exception might we be catching here again?

Reply:
It _should_ just be `NotImplementedError` but in the case the metadata conversion goes wrong, I think each reader can have it's own error handling during conversion.